contrib/holland-commvault/holland_commvault/commvault.py

```diff
@@ -41,6 +41,13 @@ def main():
 
     argv = sys.argv[1:]
 
+    holland_conf = ''
```

Comment:
Why not use the same logic as in the other files?
i.e. set holland_conf and then prepend /usr/local for freebsd